src/tasks/task1.js

```diff
@@ -0,0 +1,28 @@
+let _chunk;
+const _reverse = str => {
+  return str.split('').reverse((a, b) => {
```

Comment:
why do you use comparator like function as parameter of Array.prototype.reverse?

Reply:
Have discussed it.